numpy/tests/test_lazyloading.py

```diff
@@ -0,0 +1,37 @@
+import sys
+import importlib
+from importlib.util import LazyLoader, find_spec, module_from_spec
+import pytest
+
+
+@pytest.mark.filterwarnings("ignore:The NumPy")
```

Comment:
This is a bit unclear, the full warning is:
```
>       core.multiarray._multiarray_umath._reload_guard()
E       UserWarning: The NumPy module was reloaded (imported a second time). This can in some cases result in small but subtle issues and is discouraged.
```
It's coming from `numpy/__init__.py` which has:
```python
    # Give a warning if NumPy is reloaded or imported on a sub-interpreter
    # We do this from python, since the C-module may not be reloaded and
    # it is tidier organized.
    core.multiarray._multiarray_umath._reload_guard()
```

Can you make this
```
# Warning raised by _reload_guard() in numpy/__init__.py
@pytest.mark.filterwarnings("ignore:The NumPy module was reloaded")
```
so it's clear what this is for?